frontend/src/libs/components/form/libs/helpers/handle-server-error.helper.ts

```diff
@@ -8,7 +8,7 @@ import { ServerErrorType } from '~/libs/enums/enums.js';
 import { type HttpError } from '~/libs/packages/http/http.js';
 import { type FormField } from '~/libs/types/types.js';
 
-import { FIELD_PATH_DELIMITER } from '../consts/consts.js';
+import { PATH_DELIMITER } from '../consts/consts.js';
```

Comment:
I'm guessing it's supposed to be constants?
```suggestion
import { PATH_DELIMITER } from '../constants/constants.js';
```